src/resources/views/fields/date.blade.php

```diff
@@ -3,7 +3,7 @@
 <?php
 // if the column has been cast to Carbon or Date (using attribute casting)
 // get the value as a date string
-if (isset($field['value']) && ( $field['value'] instanceof \Carbon\Carbon || $field['value'] instanceof \Jenssegers\Date\Date )) {
+if (isset($field['value']) && ( $field['value'] instanceof \Carbon\CarbonInterface || $field['value'] instanceof \Jenssegers\Date\Date )) {
```

Comment:
Just wondering, \Jenssegers\Date\Date use is still officially supported in backpack?

Also on a side note:

https://github.com/jenssegers/date/blob/master/src/Date.php

\Jenssegers\Date\Date is a subclass of Carbon

While Carbon implements the CarbonInterface interface.

That means \Jenssegers\Date\Date is considered as an "instance" of CarbonInterface too.

Assuming you never call any \Jenssegers\Date\Date specific methods that is.

Reply:
Totally. Time to remove \Jenssegers\Date since it's no longer a package requirement. More so if it extends Carbon. Done!